### PR TITLE
make `examples.md` symlink to examples README

### PR DESCRIPTION
**Pull request by mschrimpf**
Make the examples README.md actually show up in the docs (like https://github.com/brain-score/brain-score/tree/master/docs/source).

`ln -s ../../examples/README.md examples.md`

### CODE REVIEW

**Review by kvfairchild — APPROVED** [review]

**Comment by mschrimpf** [comment]
will merge #76 into this for fixes once it's approved

**Pull request merged**
